style/swift/sample.swift

```diff
@@ -36,6 +36,15 @@ let cool = yTown(5) { foo in
   }
 }
 
+// Strongify weak references in async closures
+APIClient.getAwesomeness { [weak self] result in
+  guard let `self` = self else { return }
+  if let awesome = result.value {
```

Comment:
And I'm thinking "You know what would be so yummy here? A switch statement. Sprinkle some error handling up ins."

So maybe just some really bland uses of self like:
self.stopLoadingSpinner()
self.presentAlert()
self.updateDisplay()
self.boopBoop()